public/js/index.js

```diff
@@ -66,3 +67,9 @@ function ClipBoard(result) {
 	navigator.clipboard.writeText(result);
 	document.getElementById("copy").innerHTML='Copied'
 }
+
+function updateQRImage(shortenedUrl){
+	let QRImg = document.getElementById('QRImage');
```

Comment:
It's a good practice to use const. So replace let with const

Reply:
Yeah, you are right!

Reply:
Fixed in https://github.com/arjungarg07/Brief-Me/pull/25/commits/96f2efcd57ac7bbde8711562f84f32763563df19